lib/services.js

```diff
@@ -21,6 +21,10 @@ const apm = require('elastic-apm-node').start(),
     status = require('./middleware/status'),
     memoryStore = require('memorystore');
 
+if(process.env.ELASTIC_APM_SERVER_URL) {
+    apm.start();
+}
+
 // Load environment variables using a local .env file (see: https://www.npmjs.com/package/dotenv)
 dotEnv.config({path: path.join(process.cwd(), '.env')});
```

Comment:
If you are using dotEnv , then the if( process.env.ELASTIC_APM_SERVER_URL) should be after this condition